lib/backend/k8s/k8s.go

```diff
@@ -509,6 +509,7 @@ func buildCRDClientV1(cfg rest.Config) (*rest.RESTClient, error) {
 		if err != nil {
 			log.WithError(err).Fatal("failed to add calico resources to scheme")
 		}
+		metav1.AddToGroupVersion(scheme.Scheme, schema.GroupVersion{Group: "crd.projectcalico.org", Version: "v1"})
```

Comment:
What is this added line for?  It doesn't seem obviously connected with the rest of the change.

Reply:
I  was getting errors from the newly introduced list pager about improper registration that led me down some github issues which suggested this as the fix.

Errors looked like this:

> v1.ListOptions is not suitable for converting to "crd.projectcalico.org/v1" in scheme "pkg/runtime/scheme.go:100"